trace-forward/test/Test/Trace/Forward/Demo/Configs.hs

```diff
@@ -0,0 +1,36 @@
+module Test.Trace.Forward.Demo.Configs
+  ( mkAcceptorConfig
+  , mkForwarderConfig
+  ) where
+
+import           Control.Tracer (nullTracer)
+import           Data.IORef (IORef)
+
+import           Trace.Forward.Configuration
+import           Trace.Forward.Protocol.Type
+
+import           Test.Trace.Forward.Protocol.TraceItem
+
+mkAcceptorConfig
+  :: HowToConnect
+  -> IORef Bool
+  -> AcceptorConfiguration TraceItem
+mkAcceptorConfig ep weAreDone = AcceptorConfiguration
+  { acceptorTracer    = nullTracer
+  , forwarderEndpoint = ep
+  , whatToRequest     = GetTraceObjects 10
+  , actionOnReply     = const $ pure ()
+  , shouldWeStop      = weAreDone
+  , actionOnDone      = putStrLn "Acceptor: we are done!"
```

Comment:
Using `putStrLn` will just pollute the command line (which is annoying for tests). It's better to use logging system for such messages.